gui/control_main.py

```diff
@@ -3265,22 +3347,22 @@ def getCurrentFOV(self):
 
     def screenXPixels2microns(self, pixels):
         fov = self.getCurrentFOV()
-        fovX = fov["x"]
+        fovX = fov["x"]*1000
```

Comment:
again, these factors of 1000 need to be extracted out somewhere.

Reply:
This code will not remain for the MD2.